bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/handler/ShellyBaseHandler.java

```diff
@@ -887,10 +933,10 @@ private void checkVersion(ShellyDeviceProfile prf, ShellySettingsStatus status)
      * @param response exception details including the http respone
      * @return true if the authorization failed
      */
-    private boolean isAuthorizationFailed(ShellyApiResult result) {
+    protected boolean isAuthorizationFailed(ShellyApiResult result) {
         if (result.isHttpAccessUnauthorized()) {
             // If the device is password protected the API doesn't provide settings to the device settings
-            logger.warn("{}: {}", thingName, messages.get("init.protected"));
+            logger.info("{}: {}", thingName, messages.get("init.protected"));
             setThingOffline(ThingStatusDetail.CONFIGURATION_ERROR, "offline.conf-error-access-denied");
```

Comment:
The log message could be removed, as the status update is already logged by the framework.

The state change originated by updateStatus() is logged to events.log. Including the status detail message.